Update choose cluster docs (#9328)
solana-labs/solana: docs/src/cli/choose-a-cluster.md

# Connecting to a Cluster

Solana maintains several different clusters with different purposes.  Each

> cluster features a Solana-owned validator that serves as an entrypoint to the

Reviewer: ```suggestion
cluster features a Solana-run gossip node that serves as an entrypoint to the
```

cluster.

## Before you Begin

 - Make sure you have first
[installed the Solana command line tools](install-solana-cli-tools.md)

## Choose a Cluster

#### Mainnet Beta
A permissionless, persistent cluster for early token holders and launch partners.
No smart contracts or inflation.
 * Tokens that are issued on Mainnet Beta are **real** SOL
 * If you have paid money to purchase/be issued tokens, such as through our
 CoinList auction, these tokens will be transferred on Mainnet Beta.
   * Note: If you are using a non-command-line wallet such as
   [Trust Wallet](../wallet/trust-wallet.md),
   the wallet will always be connecting to Mainnet Beta.

>  * Entrypoint URL for Mainnet Beta: `https://api.mainnet-beta.solana.com`

Reviewer: ```suggestion
 * Gossip entrypoint for Mainnet Beta: `mainnet-beta.solana.com:8001`
 * RPC URL for Mainnet Beta: `https://api.mainnet-beta.solana.com`
```

#### Devnet
* Devnet serves as a playground for anyone who wants to take Solana for a
test drive, as a user, token holder, app developer, or validator.
* Application developers should target Devnet.
* Potential validators should first target Devnet.
* Key differences between Devnet and Mainnet Beta:
  * Devnet tokens are **not real**
  * Devnet includes a token faucet for airdrops for application testing
  * Devnet may be subject to ledger resets
  * Devnet typically runs a newer software version than Mainnet Beta
  * Devnet may be maintained by different validators than Mainnet Beta

>  * Devnet is upgraded automatically on the last Monday of each month.

Reviewer: IMO remove, this isn't too helpful and is not totally accurate.    " Devnet typically runs a newer software version than Mainnet Beta" is sufficient IMO
```suggestion

```

>  * Entrypoint URL for Devnet: `https://devnet.solana.com`

Reviewer: ```suggestion
 * Gossip entrypoint for Devnet: `devnet.solana.com:8001`
 * RPC URL for Devnet: `https://devnet.solana.com`
```

#### Testnet (Tour de SOL Cluster)
* Testnet is where we stress test recent release features on a live
cluster, particularly focused on network performance, stability and validator
behavior.
* [Tour de SOL](../tour-de-sol/README.md) initiative runs on Testnet, where we
encourage malicious behavior and attacks on the network to help us find and
squash bugs or network vulnerabilities.
* Testnet tokens are **not real**
* Testnet may be subject to ledger resets.
* Testnet typically runs a newer software release than both Devnet and
Mainnet Beta
* Testnet may be maintained by different validators than Mainnet Beta

> * Entrypoint URL for Testnet: `https://testnet.solana.com`

Reviewer: ```suggestion
* Gossip entrypoint for Testnet: `testnet.solana.com:8001`
* RPC URL for Testnet: `https://testnet.solana.com`
```
